knip.production.jsonc

```diff
@@ -1,9 +0,0 @@
-{
-	"$schema": "https://unpkg.com/knip@latest/schema.json",
-	"entry": ["src/**/index.ts!"],
-	"ignore": [
-		// https://github.com/JoshuaKGoldberg/ts-api-utils/issues/343
-		"src/usage/scopes.ts"
-	],
-	"project": ["src/**/*.ts!", "!src/test/**/*.ts!", "!src/**/*.test.ts!"]
```

Comment:
I assume knip now automatically ignores test files. Is that the case?

Reply:
Yes, in production mode, the regular entry files of plugins (Vitest in this case) are not included.